docs/specs/2fa-grace-period.md

```diff
@@ -69,15 +69,25 @@ type Mutation {
 }
 ```
 
-It's stored in the user model as `mfa_grace_period_end_at`.
+Both `setMFAGracePeriod` and `removeMFAGracePeriod` mutations are indempotent, users can have grace period granted/revoked multiple times even with same value.
+
+#### Error Response
+
+|Description|Name|Reason|Info|
+|---|---|---|---|
+|Invalid Grace Period e.g. in the past|`Invalid`|`GracePeriodInvalid`|`endAt` must be in the future.|
```

Comment:
The reason should be `MFAGracePeriodInvalid`.

The info column specifies the actual `info` object we will include. For this error, we will not include any info.

The description should also mention that the error is be returned if the given input is too far in the future. > 90 days is considered as far.

Reply:
Updated.